docs/index.md

```diff
@@ -1547,6 +1550,69 @@ The "HTML" reporter is what you see when running Mocha in the browser.  It looks
 
 [Mochawesome](https://www.npmjs.com/package/mochawesome) is a great alternative to the default HTML reporter.
 
+## Desktop Notification Support
+
+Desktop notifications allow asynchronous communication of events without
+forcing you to react to a notification immediately. Their appearance
+and specific functionality vary across platforms. They typically disappear
+automatically after a short delay, but their content is often stored in some
+manner that allows you to access past notifications.
+
+[Growl][] was an early notification system implementation for OS X and Windows,
+hence, the name of Mocha's `--growl` option.
+
+Once enabled, when your root suite completes test execution, a desktop
+notification should appear informing you whether your tests passed or failed.
+
+### Node-based notifications
+
+In order to use desktop notifications with the command-line interface (CLI),
+you **must** first install some platform-specific prerequisite software.
+Instructions for doing so can be found [here](https://github.com/mochajs/mocha/wiki/Growl-Notifications).
+
+Enable Mocha's desktop notifications as follows:
+
+```bash
```

Comment:
we need to normalize our fenced code blocks, because some of them are `sh` and some are `bash`.  should not be part of this PR--just mentioning it

Reply:
changed to `sh`